ports/casbin-cpp/vcpkg.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "casbin-cpp",
+  "version-semver": "1.52.0",
```

```suggestion
  "version": "1.52.0",
```
Is this port a semver library? If it is not, here should use `"version"`?